lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.spec.ts

```diff
@@ -233,6 +233,38 @@ describe('CardViewTextItemComponent', () => {
             renderChipsForMultiValuedProperties(cardViewTextItemObject, true, 3, '1.1', '2.2', '3.3');
         });
 
+        it('should only render new chip when provided value is valid for specified validators set', async () => {
+            const cardViewTextItemObject = {
+                label: 'Text label',
+                value: [1.1, 2.2, 3.3],
+                key: 'textkey',
+                editable: true,
+                multivalued: true,
+                type: 'float'
+            };
+            component.editable = true;
+            renderChipsForMultiValuedProperties(cardViewTextItemObject, true, 3, '1.1', '2.2', '3.3');
```

Comment:
that function returns Promise so IDE complains about await, don't we need this? 
<img width="1169" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/10e00edc-4874-4ebf-86e5-d54b4433d0e3">

Reply:
Added awaits